src/components/ons/ONSAutoSuggest.svelte

```diff
@@ -0,0 +1,138 @@
+<script>
+  import { onMount } from "svelte";
+  export let labelText, id, autosuggestValue, autosuggestData, header, invertTextColor, renderError;
+  let n;
+  let inverted = invertTextColor ? "input--with-white-description" : "";
+  let inputContainer = header || renderError ? "header-input-container" : "non-header-input-container";
+  let result;
+
+  onMount(async () => {
+    const autosuggests = [...document.querySelectorAll(".ons-js-autosuggest")];
+
+    if (autosuggests.length) {
+      const Autosuggest = (
+        await import("./../../../node_modules/@ons/design-system/components/autosuggest/autosuggest")
```

Comment:
FMI, why import this here?

Reply:
TBH, I don't fully understand it, it was there in the old code so it's here now 😆